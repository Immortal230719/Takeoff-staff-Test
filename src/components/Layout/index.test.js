import React from 'react';
import { shallow } from 'enzyme';
import { Layout } from './index';

describe('Layout', () => {
  it('should render correctly', () => {
    const component = shallow(<Layout disabled className="center">Layout</Layout>);

    expect(component.prop('children')).toEqual('Layout');
    expect(component).toMatchSnapshot();
  });
});
