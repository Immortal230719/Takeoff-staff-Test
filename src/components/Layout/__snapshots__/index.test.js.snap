// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout should render correctly 1`] = `
<WithStyles(ForwardRef(Container))
  className="center"
  disabled={true}
  maxWidth="lg"
  style={
    Object {
      "position": "relative",
      "zIndex": 1,
    }
  }
>
  Layout
</WithStyles(ForwardRef(Container))>
`;
